vue/mixins/part.js

```diff
@@ -23,10 +24,16 @@ export const partMixin = {
             code = code || error.code;
             const query = { message: message || error.message };
             if (code) query.code = code;
-            this.$root.$router.push({
-                name: "error",
-                query: query
-            });
+
+            const hasErrorPage = this.$root.$router.resolve({ name: "error" }).href !== "/";
+            if (hasErrorPage) {
+                this.$root.$router.push({
+                    name: "error",
+                    query: query
+                });
+            } else {
+                this.alertSimple(error.message);
+            }

```

Comment:
Keep the original implementation, let's force clients of this to provide an error page